include/boost/gil/extension/rasterization/circle.hpp

```diff
@@ -9,8 +9,8 @@
 #ifndef BOOST_GIL_RASTERIZATION_CIRCLE_HPP
 #define BOOST_GIL_RASTERIZATION_CIRCLE_HPP
 
-#include <boost/gil/detail/math.hpp>
-#include <boost/gil/point.hpp>
+#include "boost/gil/detail/math.hpp"
+#include "boost/gil/point.hpp"
```

Comment:
Why change the convention?

Reply:
Thanks for catching it. My guess is that IDE did that, but I don't know for sure.

Reply:
I see. Let's stick to the Boost-wide convention of `<>`

Reply:
I see. I use CLion from time to time. Good to know